### PR TITLE
Add swords

### PR DESCRIPTION
**Pull request by mn5hk**
Hi @strawpants , request you to have a look at the changes and to merge it with the main repo.

PS I am still getting my hands on with working on github in a collaborative manner x

Thanks,
AS

### CODE REVIEW

File: geoslurp_userplugins/sword.py [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+from zipfile import ZipFile
+import os
+import tempfile
+import yaml
+
+class SwordBase(OGRBase):
+    """
+    Base class for SWORD: the SWOT River Database, Version 15. If you would like more information about SWORD, please see the paper published in Water Resources Research in 2021, led by Elizabeth Altenau. You can access it here: https://agupubs.onlinelibrary.wiley.com/doi/10.1029/2021WR030054
+    """
+    import os
```
Comment (strawpants) [inline geoslurp_userplugins/sword.py:36]: Hi Amin, the `import os` and `import zipfile` are not needed here, as they are already covered above

File: geoslurp_userplugins/sword.py [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+# You should have received a copy of the GNU Lesser General Public
+# License along with geoslurp; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+
+# Author Roelof Rietbroek (r.rietbroek@utwente.nl), 2023
+# Author Amin Shakya (a.shakya@utwente.nl), 2023
+
+from geoslurp.dataset.OGRBase import OGRBase
+from geoslurp.datapull.http import Uri as http
+from geoslurp.datapull.github import Crawler as ghCrawler
```
Comment (strawpants) [inline geoslurp_userplugins/sword.py:21]: The ghcrawler and other github related imports are not needed here. I guess these are leftovers from the original file

**Review by strawpants — CHANGES_REQUESTED** [review]
Hi Amin,
looks good, for clarity, I think there are just some imports which can be removed as they are not used or double. Can you remove these before I merge the pull request?
R.

File: geoslurp_userplugins/sword.py (hunk 1 above)
Comment (mn5hk) [reply geoslurp_userplugins/sword.py]: @strawpants I have now deleted github related imports and the redundant `import os` and `import zipfile`. Request for a recheck.

Best,
AS

**Review by strawpants — APPROVED** [review]
Thanks!

**Pull request merged**